app/src/main/java/fr/free/nrw/commons/bookmarks/BookmarkFragment.java

```diff
@@ -65,12 +74,26 @@ public View onCreateView(@NonNull final LayoutInflater inflater,
     // reference to the Fragment from FragmentManager, using findFragmentById()
     supportFragmentManager = getChildFragmentManager();
 
-    adapter = new BookmarksPagerAdapter(supportFragmentManager, getContext());
+    adapter = new BookmarksPagerAdapter(supportFragmentManager, getContext(),
+                                        applicationKvStore.getBoolean("login_skipped"));
     viewPager.setAdapter(adapter);
     tabLayout.setupWithViewPager(viewPager);
+    setupTabLayout();
     return view;
   }
 
+  /**
+   * This method sets up the tab layout.
+   * If the adapter has only one element it sets the visibility of tabLayout to gone.
+   */
+  public void setupTabLayout(){
+    tabLayout.setVisibility(View.VISIBLE);
+    if(adapter.getCount()==1){
```

Comment:
We usually put spaces before and after `==` and other logical operators, see for instance the code snippets found at http://www.literateprogramming.com/javaconv.pdf